component/src/main/java/com/synopsys/integration/alert/component/settings/convert/ProxyConfigurationModelConverter.java

```diff
@@ -42,32 +43,32 @@ public ProxyConfigurationModelConverter(SettingsProxyValidator validator) {
 
     @Override
     public Optional<SettingsProxyModel> convert(ConfigurationModel globalConfigurationModel) {
-        Optional<SettingsProxyModel> convertedModel = Optional.empty();
-        SettingsProxyModel model = new SettingsProxyModel();
-        model.setName(AlertRestConstants.DEFAULT_CONFIGURATION_NAME);
-        try {
```

Comment:
Since proxy port must be handled prior to creating the model we no longer need the try-catch block